docs/view_components/scenarios.rst

```diff
@@ -0,0 +1,36 @@
+.. reference for this view-component
+.. you can refer to this component using :ref:`<component_name>-label`
+
+.. _scenarios-label:
+
+
+Scenarios
+---------
+* This component depends a lot on the tool mechanism which has not been discussed deeply yet.
+* Each project consists of a model that is run in a simulation following a specific scenario.
+
+Attributes
+^^^^^^^^^^
+*This section will be filled automatically with the input/output excel sheet, here is one example:*
+
+* Source power mix: Percentage of each source of power (inclusive % from Grid)
+
+Actions
+^^^^^^^
+* Source power mix: Input percentage of each source of power
+
+
+Requirement
+^^^^^^^^^^^
+
+*Source power mix: We should determine for each attribute if it is a requirement or optional for the simulation (depends on the tool mechanism)*
+
+Link with other view-components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Each Source power mix depends on the technology put in the model by the user
```

Comment:
This info is in the ref:`es-network-label` view-component. You can simply indicate to which other view components this view-component is linked

As I write that, I realize we have different categories of links:
- depends on attribute of other view-component
- can display other view-component from this view-component
- changes on this view-component will affect other view components

In order to be able to map these dependency (with an image) later on, I think we should format it the following way:

```
Link with other view-components
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

<reference to other component>:
    description or additional text

<reference to another component>:
    description or additional text

```

Note there should be no empty line between the <reference to another component> and the description, there is an indent before the description and there must be a blank line after the description (for proper rendering)